lib/src/form_builder_validators.dart

```diff
@@ -1,6 +1,9 @@
+import 'dart:io';
```

Comment:
Would be work in web platform? https://api.flutter.dev/flutter/dart-io/dart-io-library.html